djam_allauth_client/views.py

```diff
@@ -8,7 +8,15 @@
 from rest_framework import status
 from django.conf import settings
 
-from djam_allauth_client.provider import DJAM_SESSION_COOKIE_NAME
+from djam_allauth_client.provider import DJAM_SESSION_COOKIE_NAME, DJAM_SESSION_TOKEN_COOKIE
+
+
+class DjamCallbackView(OAuth2CallbackView):
+
+    def dispatch(self, request, *args, **kwargs):
+        response = super(DjamCallbackView, self).dispatch(request, *args, **kwargs)
+        response.set_cookie(DJAM_SESSION_TOKEN_COOKIE, domain='localhost',value= request.session.get('session_token'))
```

Comment:
you could have the domain taken from the settings, and not hard coding in as `domain='localhost'` - it could cause some troubles when deploying this :P

Reply:
damn i forget to remove this...